static/img/logo.svg

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
+<svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0" y="0" width="400" height="400" viewBox="0, 0, 400, 400">
+  <g id="Layer_1">
+    <g>
+      <path d="M57.468,302.66 L43.092,294.36 L203.242,16.98 L217.618,25.28 z" fill="#E434AA"/>
+      <path d="M39.8,272.2 L360.1,272.2 L360.1,288.8 L39.8,288.8 z" fill="#E434AA"/>
+      <path d="M206.348,374.026 L46.138,281.526 L54.438,267.15 L214.648,359.65 z" fill="#E434AA"/>
+      <path d="M345.522,132.947 L185.312,40.447 L193.612,26.071 L353.822,118.571 z" fill="#E434AA"/>
+      <path d="M54.482,132.883 L46.182,118.508 L206.392,26.008 L214.692,40.383 z" fill="#E434AA"/>
+      <path d="M342.568,302.663 L182.418,25.283 L196.794,16.983 L356.944,294.363 z" fill="#E434AA"/>
+      <path d="M52.5,107.5 L69.1,107.5 L69.1,292.5 L52.5,292.5 z" fill="#E434AA"/>
+      <path d="M330.9,107.5 L347.5,107.5 L347.5,292.5 L330.9,292.5 z" fill="#E434AA"/>
+      <path d="M203.522,366.999 L196.272,354.442 L335.611,273.992 L342.861,286.549 z" fill="#E434AA"/>
+      <path d="M369.5,297.9 C359.9,314.6 338.5,320.3 321.8,310.7 C305.1,301.1 299.4,279.7 309,263 C318.6,246.3 340,240.6 356.7,250.2 C373.5,259.9 379.2,281.2 369.5,297.9" fill="#E434AA"/>
+      <path d="M90.9,137 C81.3,153.7 59.9,159.4 43.2,149.8 C26.5,140.2 20.8,118.8 30.4,102.1 C40,85.4 61.4,79.7 78.1,89.3 C94.8,99 100.5,120.3 90.9,137" fill="#E434AA"/>
+      <path d="M30.5,297.9 C20.9,281.2 26.6,259.9 43.3,250.2 C60,240.6 81.3,246.3 91,263 C100.6,279.7 94.9,301 78.2,310.7 C61.4,320.3 40.1,314.6 30.5,297.9" fill="#E434AA"/>
+      <path d="M309.1,137 C299.5,120.3 305.2,99 321.9,89.3 C338.6,79.7 359.9,85.4 369.6,102.1 C379.2,118.8 373.5,140.1 356.8,149.8 C340.1,159.4 318.7,153.7 309.1,137" fill="#E434AA"/>
+      <path d="M200,395.8 C180.7,395.8 165.1,380.2 165.1,360.9 C165.1,341.6 180.7,326 200,326 C219.3,326 234.9,341.6 234.9,360.9 C234.9,380.1 219.3,395.8 200,395.8" fill="#E434AA"/>
+      <path d="M200,74 C180.7,74 165.1,58.4 165.1,39.1 C165.1,19.8 180.7,4.2 200,4.2 C219.3,4.2 234.9,19.8 234.9,39.1 C234.9,58.4 219.3,74 200,74" fill="#E434AA"/>
+    </g>
+  </g>
+</svg>
```

Comment:
Did you use the logo from here: https://graphql.org/brand/ ?
Just checking to be sure that we all using the same logo.